vue2/app/src/views/LoaderPage.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <div>
+      <h1>Loaders:</h1>
+      <div>
+        <p>Loader без параметров</p>
+        <Loader></Loader>        
+        <br>
+        <br>
+        <Loader loaderStyle="red"></Loader>
```

Comment:
The loader is not a button with logic for appending content to document, loader this is global function on $root element for appending content. Think about that

Reply:
I do understand that loader should be basically a method, which could be accessed from any component, and i could do that, but then we get another problem - usage of store. If we approach loader like that - we do not need store at all. For the sake of complete Task 3 I had to make some assets, so i made my Loader a button with values, we get from store (or server for task4).

Reply:
The store needs only for saving data in labwork 3. This is not practice real, just only for your task, for you exp